### ISSUE TITLE
[Feature Request] Put the COMMS part

### ISSUE THREAD

**Issue by crashzk** [TF2Romania/SourceBans-CS-GO-Theme#1]
You can see in the version of the standard SourceBans theme that has the COMMS part, follow the link to verify: https://zkservidores.com/sourcebans/index.php

In its theme does not have the part of COMMS, could add, I believe in the version of TF2 also does not have.

**Comment by MikesTooLz** [comment]
Yes, I'm having the same issue. Comms works on default theme but not on this great CSGO them.

Both the Comms page dont work as well as the option to enable comms is missing from the settings page.

**Comment by crashzk** [comment]
Exact @MikesTooLz, in case you need @TF2Romania to update it, or someone knowledgeable to upgrade.

I will be waiting to see if anyone updates, I really liked this theme.

**Comment by MikesTooLz** [comment]
The author posted this on alliedmods and hasn't responded since 2015. I don't think we will be getting any response from them.

**Comment by MikesTooLz** [comment]
I was able to get the theme working, It seemed to be missing some files. here is how to fix.

Browse to the cs-go theme folder and delete all files/folders inside of it.
Now copy all files/folders from the Default theme and place them into the cs-go folder.
The last step is to copy the cs-go theme files from this repository into your cs-go them folder overwriting any existing files.

**Comment by crashzk** [comment]
I did this too, so far no problem.
